src/components/Quiz.jsx

```diff
@@ -1,27 +1,30 @@
 export default function Quiz(props) {
-
     const answerOptions = props.choices.map((choice, index) => {
         function answerClass() {
             if (props.showResults && choice === props.correctAnswer) {
                 return "correct-answer"
             }
-            if (props.showResults && props.selectedAnswer === choice) {
+            if (props.showResults && choice === props.selectedAnswer) {
```

Comment:
I switched the order of these values so they match the order used in the check for the correct answer above.